README.md

```diff
@@ -1 +1,70 @@
-# math-magicians
\ No newline at end of file
+# Getting Started with Create React App
```

Comment:
- [ ] Kindly make your README file descriptive using the [README template](https://github.com/microverseinc/readme-template/blob/master/README.md)